crates/iota-sdk-transaction-builder/src/builder/mod.rs

```diff
@@ -1184,19 +1185,15 @@ impl<C: ClientMethods, L> TransactionBuilder<C, L> {
     /// finalization.
     pub async fn execute_with_sponsor(
         mut self,
-        keypair: &SimpleKeypair,
-        sponsor_keypair: &SimpleKeypair,
+        keypair: &impl Signer,
+        sponsor_keypair: &impl Signer,
```

Comment:
Any reason this wasn't changed in the FFIs?

Reply:
done